src/node/cli.ts

```diff
@@ -176,31 +156,18 @@ const options: Options<Required<Args>> = {
   "user-data-dir": { type: "string", path: true, description: "Path to the user data directory." },
   "extensions-dir": { type: "string", path: true, description: "Path to the extensions directory." },
   "builtin-extensions-dir": { type: "string", path: true },
-  "extra-extensions-dir": { type: "string[]", path: true },
-  "extra-builtin-extensions-dir": { type: "string[]", path: true },
   "list-extensions": { type: "boolean", description: "List installed VS Code extensions." },
   force: { type: "boolean", description: "Avoid prompts when installing VS Code extensions." },
-  "install-source": { type: "string" },
   "locate-extension": { type: "string[]" },
   "install-extension": {
     type: "string[]",
     description:
       "Install or update a VS Code extension by id or vsix. The identifier of an extension is `${publisher}.${name}`.\n" +
       "To install a specific version provide `@${version}`. For example: 'vscode.csharp@1.2.3'.",
   },
-  "enable-proposed-api": {
-    type: "string[]",
-    description:
-      "Enable proposed API features for extensions. Can receive one or more extension IDs to enable individually.",
-  },
   "uninstall-extension": { type: "string[]", description: "Uninstall a VS Code extension by id." },
   "show-versions": { type: "boolean", description: "Show VS Code extension versions." },
   "proxy-domain": { type: "string[]", description: "Domain used for proxying ports." },
-  "ignore-last-opened": {
```

Comment:
Removing this would cause a regression in behavior right?

Reply:
AFAIK this argument isn’t referenced anywhere in code-server or upstream.

Reply:
Oh snap that means we lost it so definitely a regression.